firewall.conf

```diff
@@ -26,6 +24,20 @@ iptables -A INPUT -p tcp --dport 4380:4400 -j ACCEPT   # nginx -> Google App Eng
 iptables -A INPUT -p tcp --dport 17443 -j ACCEPT       # AppController
 iptables -A INPUT -p tcp --dport 4343 -j ACCEPT        # UserAppServer
 
+# Dashboard
+# Note: If you are using Shibboleth for authentication, comment out the
+# following two lines and use the subsequent six lines instead.
+iptables -A INPUT -p tcp --dport 1080 -j ACCEPT
+iptables -A INPUT -p tcp --dport 1443 -j ACCEPT
+
+# Only allow connections to the dashboard from the Shibboleth connector.
+# APPSCALE_DNS=dns-for-your-appscale-instance.com
+# SHIBBOLETH_CONNECTOR=your-shibboleth-connector.com
+# iptables -A INPUT -s ${APPSCALE_DNS}  -p tcp --dport 1080 -j ACCEPT
```

Comment:
Perhaps you can do something line APPSCALE_NDS="-s dns-..." and have the iptables active all the time: if DNS is uncommented, it will be active only toward the appscale instance, otherwise all will be able to access the dashboard.